.github/workflows/update-insecure-dependencies.yaml

```diff
@@ -55,7 +53,22 @@ jobs:
       - name: "Update dependencies"
         id: update
         run: |
-          osv-scanner --lockfile=go.mod --json | jq '.results[].packages[].package.name' | grep -Ev ${{ env.IGNORE_PACKAGES }} | xargs -I {} go get -u {}
+          for dep in $(osv-scanner --lockfile=go.mod --json | jq -c '.results[].packages[] | .package.name as $vulnerablePackage | {
```

Comment:
nitpick: can we move it to a file inside the repo and run this via make? One line is easy to re-run locally, now it's growing so I think it would be better when it's a separate script.

Reply:
could be better to have a make target yeah